Test failure: production_pause_vote_timeout

https://github.com/AntelopeIO/spring/actions/runs/15765230461/job/44440618714

```
Traceback (most recent call last):
  File "/__w/spring/spring/build/tests/production_pause_vote_timeout.py", line 174, in <module>
    assert node0.waitForLibToAdvance(), "node0 did not advance LIB"
AssertionError: node0 did not advance LIB
```

Looks like LIB did advance, right after timeout. Seems like we should just give it a bit more time.

--- a/tests/production_pause_vote_timeout.py
+++ b/tests/production_pause_vote_timeout.py
@@ -171,7 +171,8 @@ try:
     centerNode.relaunch()
 
     Print("Verify production unpaused and LIB advances after restart of centerNode")
-    assert node0.waitForLibToAdvance(), "node0 did not advance LIB"
+    # large timeout as it can take a while for votes to catchup and LIB to advance
+    assert node0.waitForLibToAdvance(timeout=60), "node0 did not advance LIB"
     assert node1.waitForLibToAdvance(), "node1 did not advance LIB"
     assert producercNode.waitForLibToAdvance(), "producercNode did not advance LIB"
     assert producercNode.processUrllibRequest("producer", "paused", returnType=ReturnType.raw) == b'false', "producercNode should have resumed production after centerNode restarted"

